src/components/clusters/NewClusterPage.tsx

```diff
@@ -130,6 +132,11 @@ const NewClusterForm: React.FC<NewClusterFormProps> = ({ pullSecret = '', versio
                       </Text>
                     </TextContent>
                     <InputField ref={nameInputRef} label="Cluster Name" name="name" isRequired />
+                    <SingleNodeCheckbox
```

Comment:
This checkbox needs to be rendered only on standalone. We don't want to expose this option in OCM yet. 

```
  if (!ocmClient) {
...
```